Makefile

```diff
@@ -43,6 +42,11 @@ ALL_SRC = $(shell find . \( -name '*.go' -o -name '*.sh' -o -name 'Makefile*' \)
 				   -type f | \
 				sort)
 
+# All .sh or .py or Makefile or .mk files that should be auto-formatted and linted.
+SCRIPTS_SRC = $(shell find . \( -name '*.sh' -o -name '*.py' -o -name 'Makefile*' \) \
```

Comment:
Also finds the files in submodules: `jaeger-ui`, `idl`, `idl/opentelemetry-proto`. Is that behaviour appropriate?

Reply:
no, we don't want to go to submodules, need to exclude them.